Ryujinx.Memory/Range/MultiRange.cs

```diff
@@ -75,6 +76,51 @@ public MultiRange(MemoryRange[] ranges)
             }
         }
 
+        /// <summary>
+        /// Gets a slice of the multi-range.
+        /// </summary>
+        /// <param name="offset">Offset of the slice into the multi-range in bytes</param>
+        /// <param name="size">Size of the slice in bytes</param>
+        /// <returns>A new multirange representing the given slice of this one</returns>
```

Comment:
```suggestion
        /// <returns>A new multi-range representing the given slice of this one</returns>
```
To match the summary.